packages/web-app-files/tests/unit/components/Upload/DetailsWidget.spec.js

```diff
@@ -0,0 +1,77 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import DesignSystem from 'owncloud-design-system'
+
+import DetailsWidget from '@files/src/components/Upload/DetailsWidget.vue'
+
+const localVue = createLocalVue()
+localVue.use(DesignSystem)
+
+const selectors = {
+  oCProgress: 'oc-progress-stub',
+  itemName: '#item-name',
+  itemSize: '#item-size',
+  progressText: '.oc-progress-text'
+}
+
+const items = {
```

Comment:
```suggestion
const item = {
```
looks like a single item